README.md

```diff
@@ -73,6 +73,26 @@ For a more lightweight, and scriptable alternative, there is
 [spotifyd-http](https://github.com/Spotifyd/spotifyd-http), which is a work in
 progress but already supports basic tasks.
 
+## Running as a systemd service
+
+A systemd.service unit file is provided to help run spotifyd as a service on
+systemd-based systems. The file `contrib/spotifyd.service` should be copied to
+either:
+
+    cd /etc/systemd/user/
+    cd ~/.config/systemd/user/
+
+Distributions packagers of systemd-based distributions are encouraged to
+include the file downstream packagers in the former location. End-user will
```

Comment:
Both "Distributions packagers" and "downstream packagers" is confusing here. Perhaps just "Packagers are encouraged to include the file in the former location."